elements/pfe-progress-indicator/README.md

```diff
@@ -0,0 +1,44 @@
+# PFElements Progress Indicator Element
+
+`pfe-progress-indicator` is a call-to-action (CTA) element, that stands out from regular hypertext links, and is used for linking users to webpages.
```

Comment:
This description seems potentially out of date?  Is it considered a call-to-action?